### ISSUE TITLE
Bug in model checkpoint names

### ISSUE THREAD

**Issue by f-hafner** [odissei-lifecourse/life-sequencing-dutch#49]
the validation loss does not show correctly in the model names:
`model-epoch=XX-step=XXX-val_loss=0.00.ckpt` -> 0.00 should be replaced with the actual validation loss

**Comment by tanzir5** [comment]
Also the version names should be fixed.

**Comment by f-hafner** [comment]
has this been fixed, @enamcse?

**Comment by enamcse** [comment]
Yes, as far as I observed, it shows correctly. Though sometimes the step count shows ± 1 especially for resuming from a checkpoint but that's a different issue, I believe.

**Comment by f-hafner** [comment]
ok, thanks, I'll close this then.
